modules/rostests/apitests/ntdll/NtRaiseHardError.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for NtRaiseHardError
+ * COPYRIGHT:   Copyright 2017 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+#include <apitest.h>
+
+#define WIN32_NO_STATUS
+#include <ndk/exfuncs.h>
+#include <strsafe.h>
+#include <tlhelp32.h>
+
+
+START_TEST(NtRaiseHardError)
+{
+    PROCESS_INFORMATION pi;
+    STARTUPINFOA si = { 0 };
+    char self[MAX_PATH];
+    char buffer[MAX_PATH*2];
+    int argc;
+    char** argv;
+    BOOL ret;
+    DWORD dwret;
+    HANDLE hDup;
+
+    //if (!winetest_interactive)
+    //{
+    //    skip("Spawns dialogs that need to be manually closed (set WINETEST_INTERACTIVE=1)\n");
+    //    return;
+    //}
+
+    argc = winetest_get_mainargs(&argv);
+    if (argc >= 3)
+    {
+        if (!strcmp(argv[2], "raise"))
+        {
+            ULONG HardErrorResponse;
+            SetErrorMode(0);
+            NtRaiseHardError(STATUS_INVALID_IMAGE_FORMAT, 0, 0, NULL, OptionOk, &HardErrorResponse);
+            ok(0, "Child process did not die\n");
+        }
+        else
+        {
+            ok(0, "Unexpected command %s\n", argv[2]);
+        }
+
+        ExitProcess(winetest_get_failures());   /* Do not print the success / failure line */
```

Comment:
Should you `fflush(stdout)` or something first?

Reply:
Stating did not like this, since it does not correctly transfer todo's, so I might drop it entirely